Board.py

```diff
@@ -1,11 +1,163 @@
-from checker import Checker
+from enum import Enum
+
+ROWS = 8
+COLS = 8
+
+
+class Player(Enum):
+    player1 = 0
+    player2 = 1
+
+
+class Piece(Enum):
+    basic = 0
+    king = 0
+
+
+def in_bounds(x, y):
+    return 0 <= x < COLS and 0 <= y < ROWS
+
+
+def is_jump(start_x, end_x):
```

Comment:
this function title is not very descriptive, rename to `is_jump_move`. `in_bounds` is also not fully optimal for a similar reason -- `is_in_bounds` would be a better function name since it is clear that a boolean is returned from the name

Reply:
if you can't figure out a way to make the function name descriptive, add docstrings (comments) so the reader can understand what the function does